packages/gatsby-theme-newrelic/src/components/Navigation.js

```diff
@@ -0,0 +1,30 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import NavigationContext from './NavigationContext';
+
+const sanitizeSearchTerm = (searchTerm) =>
+  searchTerm.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+
+const Navigation = ({ className, children, searchTerm }) => {
+  const value = useMemo(
+    () => ({ searchTerm: sanitizeSearchTerm(searchTerm) }),
+    [searchTerm]
+  );
+
+  return (
+    <NavigationContext.Provider value={value}>
+      <nav role="navigation" aria-label="Navigation" className={className}>
+        {children}
+      </nav>
+    </NavigationContext.Provider>
+  );
+};
+
+Navigation.propTypes = {
+  children: PropTypes.node,
```

Comment:
Since we have it listed as required in the docs, we should probably update it here (or just update the docs).

```suggestion
  children: PropTypes.node.isRequired,
```

Reply:
Oops, meant to check that and didn't. Nice catch!